tools/common/Application.cs

```diff
@@ -899,6 +899,42 @@ public void ParseAbi (string abi)
 			abis = res;
 		}
 
+		public void ParseRegistrar (string v)
+		{
+			var split = v.Split ('=');
+			var name = split [0];
+			var value = split.Length > 1 ? split [1] : string.Empty;
+			switch (name) {
+			case "static":
+				Registrar = RegistrarMode.Static;
+				break;
+			case "dynamic":
+				Registrar = RegistrarMode.Dynamic;
+				break;
+			case "default":
+				Registrar = RegistrarMode.Default;
+				break;
+			case "partial":
+			case "partial-static":
+				Registrar = RegistrarMode.PartialStatic;
```

Comment:
what will happen inside `mtouch` ?

Reply:
I don't know, so I added an ifdef to avoid the problem.